session.go

```diff
@@ -74,17 +87,32 @@ func (session *Session) ExecuteWithParameter(stmt string, params map[string]inte
 		}
 		return resSet, nil
 	}
-
 	resp, err := session.executeWithReconnect(execFunc)
 	if err != nil {
 		return nil, err
 	}
 	return resp.(*ResultSet), err
 }
 
-// Execute returns the result of the given query as a ResultSet
-func (session *Session) Execute(stmt string) (*ResultSet, error) {
-	return session.ExecuteWithParameter(stmt, map[string]interface{}{})
+func (session *Session) ExecuteWithTimeout(stmt string, timeoutMs int64) (*ResultSet, error) {
+	return session.ExecuteWithParameterTimeout(stmt, map[string]interface{}{}, timeoutMs)
+}
+
+func (session *Session) ExecuteWithParameterTimeout(stmt string, params map[string]interface{}, timeoutMs int64) (*ResultSet, error) {
+	session.mu.Lock()
+	defer session.mu.Unlock()
+	if timeoutMs <= 0 {
```

Comment:
how about keep consistent with timeout in config？
`if timeoutMs <0, then use the default value 0.` And when timeout is 0, it means there's no timeout.  We can set it to Max Value.
https://github.com/vesoft-inc/nebula-go/blob/1c445dca8a76dbcc64e5ec082ec1e44199b5c490/configs.go#L42C1-L50C3

Reply:
if timeout is max value, there's no any timeout and the performance is poorer then execute directly.

prefer raise an error, and user can use execute directly or correct the timeout value.

Reply:
that is to say.
0 is useful for connection timeout, but is not for execute with timeout.